desktop/apps/consignments2/components/thank_you/index.js

```diff
@@ -8,9 +8,9 @@ import block from 'bem-cn'
 import { connect } from 'react-redux'
 import { get } from 'lodash'
 
-function ThankYou ({ submission, uploadedImages }) {
+function ThankYou ({ submission, processingImages, uploadedImages }) {
   const b = block('consignments2-submission-thank-you')
-  const submissionImage = get(uploadedImages, '0.src') ? get(uploadedImages, '0.src') : '/images/missing_image.png'
+  const submissionImage = get(uploadedImages, '0.src') && processingImages.length === 0 ? get(uploadedImages, '0.src') : '/images/missing_image.png'
```

Comment:
Even though we have your image `src` from uploading it via `gemup`, we only show it here if it has actually _processed_. Otherwise it's kind of a confusing UI.